src/lambda/models/samLambdaRuntime.ts

```diff
@@ -76,15 +53,6 @@ export function getFamily(runtime: string | undefined): SamLambdaRuntimeFamily {
         case 'dotnetcore':
```

Comment:
We need to keep the generic names here for backwards compatibility with existing SAM templates since localLambdaRunner has functions that check against the runtime present in the SAM/Cloudformation template.